app/models/upload.rb

```diff
@@ -0,0 +1,9 @@
+class Upload < ActiveRecord::Base
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  has_attached_file :image, :styles => { :medium => "500x500>", :thumb => "200x200>" },
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)